webserver/views/api/v1/core.py

```diff
@@ -196,9 +196,7 @@ def get_many_lowlevel():
         try:
             recording_details.setdefault(recording_id, {})[offset] = db.data.load_low_level(recording_id, offset)
         except NoDataFoundException:
-            errors = recording_details.setdefault("errors", {})
-            mbid_entry = errors.setdefault(recording_id, {})
-            mbid_entry[offset] = {"message": "recording does not exist"}
+            pass
```

Comment:
A small thing on this commit - because you're doing two distinct things, I would have made it as two commits, to make it easier to follow what changes are being made.